src/muya/lib/contentState/pasteCtrl.js

```diff
@@ -266,6 +266,7 @@ const pasteCtrl = ContentState => {
     const text = rawText || event.clipboardData.getData('text/plain')
     let html = rawHtml || event.clipboardData.getData('text/html')
     if (!text && !html) {
+      await this.pasteImage(event)
```

Comment:
we already handle pasteImage on line 298

```js
    if (!html) {
      const file = await this.pasteImage(event)
      if (file) {
        return
      }
    }
```

So can we just delete this if statement (`if (!text && !html)`)?

Reply:
I added this check because I thought we could early-return. I would prefer to handle edge cases at the beginning but I can also restore the old behavior tomorrow.